### ISSUE TITLE
Unable to create new shortcuts

### ISSUE THREAD

**Issue by madoar** [PhoenicisOrg/phoenicis#1879]
When creating a new shortcut on the current master I get the following exception:

```
Exception in thread "pool-3-thread-2" java.lang.ClassCastException: class java.lang.String cannot be cast to class jdk.nashorn.api.scripting.ScriptObjectMirror (java.lang.String is in module java.base of loader 'bootstrap'; jdk.nashorn.api.scripting.ScriptObjectMirror is in module jdk.scripting.nashorn of loader 'platform')
	at phoenicis.javafx@5.0-SNAPSHOT/org.phoenicis.javafx.views.mainwindow.library.LibraryView.lambda$createShortcut$10(LibraryView.java:353)
	at phoenicis.scripts@5.0-SNAPSHOT/org.phoenicis.scripts.nashorn.NashornInteractiveSession.eval(NashornInteractiveSession.java:34)
	at phoenicis.scripts@5.0-SNAPSHOT/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:43)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

More details contains the following message shown inside an error dialog:

```
javax.script.ScriptException: <eval>:1:40 Missing close quote
include("engines.wine".shortcuts.wine");
                                        ^ in <eval> at line number 1 at column number 40
	at jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:477)
	at jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:544)
	at jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:531)
	at jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:409)
	at jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:162)
	at java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
	at phoenicis.scripts@5.0-SNAPSHOT/org.phoenicis.scripts.nashorn.NashornEngine.evalAndReturn(NashornEngine.java:59)
	at phoenicis.scripts@5.0-SNAPSHOT/org.phoenicis.scripts.nashorn.NashornInteractiveSession.eval(NashornInteractiveSession.java:34)
	at phoenicis.scripts@5.0-SNAPSHOT/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:43)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: jdk.nashorn.internal.runtime.ParserException: <eval>:1:40 Missing close quote
include("engines.wine".shortcuts.wine");
                                        ^
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.Lexer.error(Lexer.java:1860)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.Lexer.scanString(Lexer.java:1006)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.Lexer.lexify(Lexer.java:1717)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.getToken(AbstractParser.java:135)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.nextToken(AbstractParser.java:216)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.nextOrEOL(AbstractParser.java:173)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.next(AbstractParser.java:160)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.scanFirstToken(Parser.java:293)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.parse(Parser.java:323)
	at jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.parse(Parser.java:285)
	at jdk.scripting.nashorn/jdk.nashorn.internal.runtime.Context.compile(Context.java:1500)
	at jdk.scripting.nashorn/jdk.nashorn.internal.runtime.Context.compileScript(Context.java:1467)
	at jdk.scripting.nashorn/jdk.nashorn.internal.runtime.Context.compileScript(Context.java:750)
	at jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:542)
	... 10 more
```

**Comment by plata** [comment]
```
include("engines.wine".shortcuts.wine");
```
should be
```
include("engines.wine.shortcuts.wine");
```

**Comment by plata** [comment]
Fixed in #1878.
